RecoTracker/LSTCore/src/alpaka/PixelTriplet.h

```diff
@@ -208,64 +208,6 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
     pixelTripletsInGPU.rzChiSquared[pixelTripletIndex] = rzChiSquared;
   };
 
-  template <typename TAcc>
-  ALPAKA_FN_ACC ALPAKA_FN_INLINE bool runPixelTrackletDefaultAlgopT3(TAcc const& acc,
```

Comment:
is this just moved down?
This will be a pain to reconcile with my pT3 SoA migration

Reply:
I'll move it back up. It stopped letting me compile it after some change I made. I'm not sure how it was compiling before since the functions are defined out of order.